docs/sources/reference/components/prometheus.remote_write.md

```diff
@@ -472,6 +474,53 @@ If the component falls behind more than one third of the data written since the
 last truncate interval, it is possible for the truncate loop to checkpoint data
 before being pushed to the remote_write endpoint.
 
+### Tuning `max_shards`
+
+The [`queue_config`](#queue_config-block) block allows you to configure `max_shards`. The `max_shards` is the maximum
+number of concurrent shards sending samples to the Prometheus-compatible remote write endpoint.
+For each shard, a single remote write request can send up to `max_samples_per_send` samples.
+
+The maximum throughput that {{< param "PRODUCT_NAME" >}} can achieve when remote writing is equal to
+`max_shards * max_samples_per_send * <1 / average write request latency>`. For example, running {{< param "PRODUCT_NAME" >}} with the
```

Comment:
Nit: any reason why not just `(max_shards * max_samples_per_send) / <average write request latency>`?

Reply:
I think it depends how you read it. I don't have a big preference. I did initially as you suggested, then received [this comment](https://github.com/grafana/alloy/pull/1018#discussion_r1634987255) from Dimitar.